src/renderers/common/Textures.js

```diff
@@ -147,7 +138,6 @@ class Textures extends DataMap {
 
 				const texture = textures[ i ];
 
-				texture.isTextureArray = renderTarget.multiview === true && size.depth > 1;
```

Comment:
I fear this removal breaks multiview. When rendering in XR the framebuffer render target is resized with a depth value larger `1`.

https://github.com/mrdoob/three.js/blob/6e564589c5bc5a374295f1218fd95b2645e577c6/src/renderers/common/Renderer.js#L1246

 But since the `isTextureArray `/`isArrayTexture` is only set in the ctor, the flag isn't correctly. So there are two option: Restore this line or make sure `Texture.isArrayTexture` is also set in `RenderTarget.setSize()` for all render target textures.

Reply:
>  make sure Texture.isArrayTexture is also set in RenderTarget.setSize() for all render target textures.
I think this option makes a lot more sense I will try.